# Namespace for this agent. If the agent_name launch arg is utilized, this must be updated,
# or none of these parameters will be grabbed. default = car1
car1:
  steering_driver:
    ros__parameters:
#      double : Lower bound in radians for servo to move, default = 0.0
      min_angle: -1.309
#      double : Upper bound in radians for servo to move, default = 0.0
      max_angle: 1.309
  throttle_driver:
    ros__parameters:
#      double : minimum throttle percentage to drive the motor, default = -1.0
      min_throttle: -1.0
#      double : maximum throttle percentage to drive the motor, default = 1.0
      max_throttle: 1.0
  bno055_driver:
    ros__parameters:
#      string: topic for IMU msg to be published, default = `imu`
      imu_topic: "imu"
#      double: rate for imu msg to be published in HZ, default = 50.0
      imu_rate: 50.0
car2:
  steering_driver:
    ros__parameters:
#      double : Lower bound in radians for servo to move, default = 0.0
      min_angle: -1.309
#      double : Upper bound in radians for servo to move, default = 0.0
      max_angle: 1.309
  throttle_driver:
    ros__parameters:
#      double : minimum throttle percentage to drive the motor, default = -1.0
      min_throttle: -1.0
#      double : maximum throttle percentage to drive the motor, default = 1.0
      max_throttle: 1.0
  bno055_driver:
    ros__parameters:
#      string: topic for IMU msg to be published, default = `imu`
      imu_topic: "imu"
#      double: rate for imu msg to be published in HZ, default = 50.0
      imu_rate: 50.0
car3:
  steering_driver:
    ros__parameters:
#      double : Lower bound in radians for servo to move, default = 0.0
      min_angle: -1.309
#      double : Upper bound in radians for servo to move, default = 0.0
      max_angle: 1.309
  throttle_driver:
    ros__parameters:
#      double : minimum throttle percentage to drive the motor, default = -1.0
      min_throttle: -1.0
#      double : maximum throttle percentage to drive the motor, default = 1.0
      max_throttle: 1.0
  bno055_driver:
    ros__parameters:
#      string: topic for IMU msg to be published, default = `imu`
      imu_topic: "imu"
#      double: rate for imu msg to be published in HZ, default = 50.0
      imu_rate: 50.0
